std/concurrency.d

```diff
@@ -963,6 +963,39 @@ bool register(string name, Tid tid)
 }
 
 /**
+Retrieves all names associated with $(LREF Tid). These names
+have all been registered with the $(LREF register) function.
+
+*/
+string[] registeredNames(Tid tid)
+{
+    synchronized (registryLock)
+    {
+        return namesByTid[tid];
+    }
+}
+
+///
+@system unittest
+{
+    __gshared string received;
+    static void spawnedFunc(Tid ownerTid)
```

Comment:
FYI: `ownerTid` can be infered and it doesn't make sense if you don't use it ;-)